onnxruntime/core/providers/cuda/cuda_graph.cc

```diff
@@ -58,7 +60,7 @@ Status CUDAGraph::Replay() {
   // Although this function is not thread safe, the lock is not needed here because
   // CUDA EP maintains a separate cuda graph per thread
 #if defined(CUDA_VERSION) && CUDA_VERSION >= 10000
-  LOGS_DEFAULT(INFO) << "Replaying CUDA graph on stream " << stream_;
+  LOGS_DEFAULT(VERBOSE) << "Replaying CUDA graph on stream " << stream_;
```

Comment:
Newly added logs are at INFO level but this is changed to VERBOSE. Was this intentional ?

Reply:
Yes, it is intentional. New logs only happen once in a session, and this log happens in each inference run, and change it to verbose to avoid logs overwhelmed by it.